csrankings-q.csv

```diff
@@ -111,6 +111,7 @@ Qingjiang Shi,Tongji University,https://sse.tongji.edu.cn/Data/View/3555,8xoKeR0
 Qingkai Zeng 0002,Nanjing University,https://cs.nju.edu.cn/58/1a/c2639a153626/page.htm,NOSCHOLARPAGE
 Qingling Duan,Queen’s University,https://dbms.queensu.ca/faculty/duan_qingling,NOSCHOLARPAGE
 Qingming Huang,Chinese Academy of Sciences,http://people.ucas.edu.cn/~0003060?language=en,J1vMnRgAAAAJ
+Qingqing Ye 0001,The Hong Kong Polytechnic University,https://www.qingqingye.net/research/,u9xzOh4AAAAJ

```

Comment:
Not in CS: https://www.polyu.edu.hk/comp/people/academic-staff/.

Reply:
Although Prof. Hu and Dr. Ye are in the Department of Electrical and Electronic Engineering of PolyU, they focus on database and security research. They frequently publish papers in top computer science conferences listed in CSRankings, such as VLDB, ICDE, S&P, SIGMOD, and NeurIPS. 

https://www.polyu.edu.hk/eee/people/academic-staff-and-teaching-staff/

Reply:
See comment above.

Reply:
See comment above. Many thanks.